src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.tsx

```diff
@@ -0,0 +1,54 @@
+import { Accordion, Heading } from '../../../../../core';
+import { useOdsModulesContext } from '../../../odsModulesProvider';
+import { ProposalActionsActionVerification } from '../proposalActionsActionVerfication/proposalActionsActionVerfication';
+import type { IProposalAction } from '../proposalActionsTypes';
+import { proposalActionsUtils } from '../utils';
+
+export interface IProposalActionsActionProps {
+    /**
+     * Proposal action
+     */
+    action: IProposalAction;
+    /**
+     * Index of the action being mapped by its parent ProposalActions.Container
+     */
+    index: number;
+    /**
+     * Toggle accordion callback for notifying the parent container to update after click
+     */
+    onToggle?: () => void;
+}
+
+export const ProposalActionsAction: React.FC<IProposalActionsActionProps> = (props) => {
+    const { action, index, onToggle } = props;
+    const ActionComponent = proposalActionsUtils.getActionComponent(action);
+    const { copy } = useOdsModulesContext();
+
+    const actionTypeToStringMapping: Record<string, string> = {
+        withdrawToken: copy.proposalActionsAction.actionTypeWithdrawToken,
+    };
+
+    if (!ActionComponent) {
+        return null;
+    }
```

Comment:
We should still render something when we don't have a specific UI for an action type

Reply:
Should I add a TODO and await a design?